oasis-webapp/src/main/resources/templates/Login.soy

```diff
@@ -67,10 +71,10 @@ small { display: block; margin-top: 2em; font-size: .65em; line-height: initial;
 aside { display: block; }
 /* Setup the aside so that it covers the bottom of the main and appears as if it was outside it. */
 aside { margin: 30px -10px -25px; } /* 30px at the top to get some space from the buttons, other margins reset main's padding. */
-aside { background: #626580; } /* Same background as the page-wrap */
+aside { background: #6f438e; } /* Same background as the page-wrap */
 
 aside>* { text-align: center; }
-aside a { display: inline-block; padding: 5px 15px; font-weight: lighter; text-decoration: none; background: #6e418e; color: white; }
+aside a { display: inline-block; padding: 5px 15px; font-weight: lighter; text-decoration: underline; background: #6f438e; color: white; }
```

Comment:
Yup. What I meant is… underline is the default styling for links, and we don't override it globally so we can just remove `text-decoration: none` instead of changing it to `underline`.